ckanext/push_errors/logging.py

```diff
@@ -12,6 +13,51 @@
 log = logging.getLogger(__name__)
 
 
+# Configuración de Redis
+redis_client = redis.StrictRedis(
+    host=toolkit.config.get('ckan.redis.url', 'redis_bcie'),
+    port=6379,
+    db=0
+)
+
+LIMIT_PER_MINUTE = 10  # Máximo de mensajes por minuto
+LIMIT_PER_HOUR = 100   # Máximo de mensajes por hora
```

Comment:
This must be settings like `ckanext.push_errors.max_message_minute` and `ckanext.push_errors.max_message_hours`

Define default values as:
 - Minute: 3 messages
 - Hour: 10 messages